.github/workflows/lint.yml

```diff
@@ -32,3 +32,17 @@ jobs:
       run: |
         git diff-index --quiet HEAD -- pwndbg tests
         ./lint.sh
+        
+    - name: Run mypy
+      uses: tsuyoshicho/action-mypy@v4
+      with:
+        github_token: ${{ secrets.TOKEN }}
```

Comment:
```suggestion
        github_token: ${{ secrets.GITHUB_TOKEN }}
```
What happens if you apply this suggestion?

Reply:
Let's see, I committed this suggestion

Reply:
I don't see the token not found error anymore, but neither do I see a review comment 😅